backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerHttpClient.java

```diff
@@ -213,7 +213,14 @@ public void cancelPlaybook(UUID uuid, int timeout) throws Exception {
         File output = File.createTempFile("output", ".log");
         String command = String.format("ansible-runner stop %1$s", uuid);
         ProcessBuilder ansibleProcessBuilder = new ProcessBuilder(command).redirectErrorStream(true).redirectOutput(output);
-        ansibleProcess = ansibleProcessBuilder.start();
+        try {
+            ansibleProcess = ansibleProcessBuilder.start();
+            // if play execution was already completed, the command fails.
+        } catch (IOException e) {
+            log.error(String.format("Failed to execute call to cancel playbook. %1$s, %2$s ",
+                    output.toString(), Paths.get(this.getJobEventsDir(uuid.toString()) + "../stdout")));
```

Comment:
Please reformat to make more readable and add full stacktrace to debug log:

```
            log.error(String.format("Failed to execute call to cancel playbook. %1$s, %2$s../stdout ",
                    output.toString(), 
                    Paths.get(this.getJobEventsDir(uuid.toString()))));
            log.debug("Exception:", e);
```

Reply:
done